packages/client/src/client/components/Line.tsx

```diff
@@ -1,8 +1,8 @@
 import styled from "styled-components"
 
-import { theme } from "../theme/uiskTheme"
+import { Spacing } from "../theme/types"
 
-export const Line = styled.div<{ height?: keyof typeof theme.spacing }>`
+export const Line = styled.div<{ height?: Spacing }>`
```

Comment:
ref - MUI uses `<hr>`
(yes, I know that is a bit odd, when vertical, but see latest RT Eq markup when I push the asset switcher bit)